docs/app/pages/getting-started/getting-started.component.html

```diff
@@ -26,42 +25,12 @@ <h2 id="pre-requisites">Pre-Requisites</h2>
                 is used to install application dependencies.
             </li>
             <li>
-                <a href="https://git-scm.com/">Git</a> &mdash; used to download the seed project, or to build UX Aspects
-                from source.
-            </li>
-            <li>
-                <a href="https://cli.angular.io/">Angular CLI</a> &mdash; used to build the seed project or create a new
+                <a href="https://cli.angular.io/">Angular CLI</a> &mdash; used to create a new
                 Angular application.
             </li>
         </ul>
     </section>
 
-    <section>
-        <h2 id="seed-project">Seed Project</h2>
-
-        <p class="m-b">
-            The easiest way to get started with UX Aspects is to download our
-            <a href="https://github.com/UXAspects/UXAspects-Seed">Seed Project</a>. This is a starter project, suitable
-            for building a new Angular application using UX Aspects. Since it is built with Angular CLI, it is easy to
-            extend with new pages and components.
-        </p>
-
-        <ol>
-            <li>
-                Clone the repository:
-                <code>git clone https://github.com/UXAspects/UXAspects-Seed.git <i>my-project</i></code>
-            </li>
-            <li>Switch to the directory which was downloaded.</li>
-            <li>Install the dependencies: <code>npm install</code></li>
-            <li>Build and run the site using the Angular CLI development server: <code>ng serve</code></li>
-        </ol>
-
-        <p>
-            A complete guide on using the Seed Project can be
-            <a href="https://github.com/UXAspects/UXAspects-Seed/blob/master/README.md">found here</a>.
-        </p>
-    </section>
-
     <section>
         <h2 id="add-to-an-existing-project">Add to an Existing Project</h2>
```

Comment:
Should we perhaps update the title here so that it is not referring to an "Existing" project? These are the steps to follow for any project now, new or existing.